backend/app/websocket.py

```diff
@@ -106,10 +106,24 @@ def process_chat_input(
             else None
         ),
         stream=True,
+        generation_config=(
```

Comment:
As well as chat.py, we don't need to handle null

Reply:
The bot can still be null.